deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
What was the failure? Those should be the only changes for undoing this scenario